ts-tests/tests/config.ts

```diff
@@ -16,3 +16,7 @@ export const BLOCK_TIMESTAMP = 6; // 6 seconds per block
 export const BLOCK_HASH_COUNT = 256;
 export const EXISTENTIAL_DEPOSIT = 10_000_000_000_000_000; // The minimum amount required to keep an account open
 export const BLOCK_GAS_LIMIT = 60000000;
+
+export const SNOW_RPC_ENDPOINT = "https://snow-rpc.icenetwork.io:9933";
```

Comment:
may be this should be configurable so that we can run these tests against other snow urls?

Reply:
The RPC url, chain id and contract address can be moved to the .env file, to make these configurable.

These values however, are not used on the github actions since they need to be run manually with `npm test-ctx-state` on every mainnet network upgrade.

Reply:
Let's add two networks Arctic and Snow. 
And possibility to run test on different networks

For example: 
```
npm run test-ctx-state arctic
npm run test-ctx-state snow

or

npm run test-ctx-state --chain arctic
npm run test-ctx-state --chain snow
```

Reply:
Source text
I think we will move these tests in to another Git Action task.(e2e tests) Because In my opinion It doesn't make any sense to run them in current flow. Because we should run this tests only after network upgrade and not for every commit.  
In this case we need to configure network in the process of running.

What do you think?